northstar_tests/tests/tests.rs

```diff
@@ -347,6 +348,50 @@ test!(connect_version, {
     runtime.shutdown().await
 });
 
+test!(connections_during_shutdown_are_ignored, {
```

Comment:
> Can me make the task break on a EventLoop closed error and return that result and assert it? Probably not because this is racy.

I did not understand this question.

> This tests asserts nothing otherwise beside that `task::spawn` result which is probably tested heavily in tokio itself.

This test asserts that it is possible to shutdown the runtime while connections are trying to start and stop containers.

Reply:
> > Can me make the task break on a EventLoop closed error and return that result and assert it? Probably not because this is racy.
> 
> I did not understand this question.

Sorry. What I wanted to ask whether it's possible to take the result of the connect and start/stop into account and assert it? Is it possible to know when we should get a ok (because we're faster than the shutdown) or a `EventLoopClosed` when the shutdown has been initiated.

Reply:
:thinking: the problem I see with that is that currently it is not easy to deterministically assert the relation between the connection attempt result and the state of the shutdown.

Reply:
Yes. From connection point of view there should be just a cannot connect or connection closed.